livekit_rtc.proto

```diff
@@ -33,6 +33,8 @@ message SignalRequest {
     SyncState sync_state = 12;
     // Simulate conditions, for client validations
     SimulateScenario simulate = 13;
+    // client triggered ping to server
+    Ping ping = 14;
```

Comment:
what about sending an int64 instead?
```suggestion
    int64 ping = 14;
```

that way when pong is received, we could compute delta on time spent.